### PR TITLE
GITC-650: inform users cap reached

### PR DESCRIPTION
**Pull request by nutrina**
##### Description

Displaying tooltips for caped grants

##### Refers/Fixes

GITC-650

##### Testing

Have tested locally, according to the acceptance criteria

### CODE REVIEW

**Comment by gdixon** [comment]
Testing:

Screenshot 2021-12-03 at 18 43 48

Screenshot 2021-12-03 at 18 57 08

File: app/assets/v2/js/cart.js [hunk 1]
```diff
@@ -257,7 +257,7 @@ Vue.component('grants-cart', {
       return {
         'token': donationToken,
         'total': totalEstimatedMatch.toFixed(2),
-        'total_str': totalEstimatedMatch.toFixed(2).toString() + ' ' + 'DAI'
+        'total_str': totalEstimatedMatch.toFixed(2).toString() + ' DAI'
```
Comment (gdixon) [inline app/assets/v2/js/cart.js:260]: This was causing eslint to fail

File: app/grants/templates/grants/cart/eth.html [hunk 2]
```diff
@@ -117,8 +117,15 @@
                     + 0 DAI
                   </span>
                   <span class="text-muted">
-                    <template v-if="grant.grant_donation_clr_match < 1000">estimated match</template>
+                    <template v-if="grant.grant_donation_clr_match < 1000">estd. match</template>
```
Comment (gdixon) [inline app/grants/templates/grants/cart/eth.html:120]: If we leave this as `estimated` then it causes an overflow

**Review by gdixon — APPROVED** [review]
Awesome job, thank you @nutrina! 🚀

**Comment by willsputra** [comment]
LGTM!

**Review by thelostone-mc — APPROVED** [review]

**Comment by frankchen07** [comment]
What happens if a grant is in two matching pools and one cap is hit and another is not? Will the tooltip still notify the user?

**Comment by gdixon** [comment]
In the situation you describe, it wouldn't be shown. We're using the summation of all clr calcs for a given grant to feed this state.

**Comment by gdixon** [comment]
In the situation you describe, it wouldn't be shown, no. 

We're using the summation of all clr calcs for a given grant to feed this state, if any matching is still available then I'm not sure it would make sense to notify the user here? Although the matching might be lower than it would have been without the cap - the users contribution would still receive matching, so if we was to show the tooltip then at the very least we would have to rethink the messaging.

I think this could be something we rework for GR13?

**Pull request merged**